Issue with cml version 0.4.1

Hey everyone! Following issue cropped up on us today when using the `cml-runner` command.

```
2021-05-20T12:48:34.4452622Z ##[group]Run iterative/setup-cml@v1
2021-05-20T12:48:34.4453084Z with:
2021-05-20T12:48:34.4453432Z   version: latest
2021-05-20T12:48:34.4453815Z ##[endgroup]
2021-05-20T12:48:35.1673922Z Uninstalling previous CML
2021-05-20T12:48:35.8848341Z Installing CML version latest
2021-05-20T12:49:00.0732031Z ##[group]Run cml-runner \
2021-05-20T12:49:00.0732660Z [36;1mcml-runner \[0m
2021-05-20T12:49:00.0733155Z [36;1m--cloud aws \[0m
2021-05-20T12:49:00.0733843Z [36;1m--cloud-region eu-west-2 \[0m
2021-05-20T12:49:00.0734364Z [36;1m--cloud-type=m \[0m
2021-05-20T12:49:00.0734907Z [36;1m--cloud-hdd-size 100 \[0m
2021-05-20T12:49:00.0735456Z [36;1m--cloud-spot \[0m
2021-05-20T12:49:00.0735933Z [36;1m--labels=cml-runner[0m
2021-05-20T12:49:00.0777864Z shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
2021-05-20T12:49:00.0778428Z env:
2021-05-20T12:49:00.0779949Z   repo_token: ***
2021-05-20T12:49:00.0780567Z   AWS_ACCESS_KEY_ID: ***
2021-05-20T12:49:00.0781481Z   AWS_SECRET_ACCESS_KEY: ***
2021-05-20T12:49:00.0781947Z   CASE_NAME: 
2021-05-20T12:49:00.0782393Z   BATCH_ID: 
2021-05-20T12:49:00.0782820Z ##[endgroup]
2021-05-20T12:49:04.0902353Z Preparing workdir /home/runner/.cml/cml-gdvla86awg...
2021-05-20T12:49:04.0907545Z Deploying cloud runner plan...
2021-05-20T12:49:04.0909813Z Terraform apply...
2021-05-20T12:49:04.0920899Z {"level":"error","status":"terminated"}
2021-05-20T12:49:04.0936814Z TypeError: Cannot read property 'replace' of undefined
2021-05-20T12:49:04.0938026Z     at run_terraform (/usr/local/lib/node_modules/@dvcorg/cml/bin/cml-runner.js:164:18)
2021-05-20T12:49:04.0939156Z     at run_cloud (/usr/local/lib/node_modules/@dvcorg/cml/bin/cml-runner.js:184:25)
2021-05-20T12:49:04.0940125Z     at run (/usr/local/lib/node_modules/@dvcorg/cml/bin/cml-runner.js:311:20)
2021-05-20T12:49:04.2383262Z [0m[1m[32m
2021-05-20T12:49:04.2384722Z Destroy complete! Resources: 0 destroyed.
2021-05-20T12:49:04.2385869Z [0
2021-05-20T12:49:04.2446233Z ##[error]Process completed with exit code 1.
```

It seems as though it's related to version `0.4.1`. Running the `setup-cml` action with version `0.4.0` seems to have fixed the problem.

--- a/bin/cml-runner.js
+++ b/bin/cml-runner.js
@@ -161,7 +161,7 @@ const run_cloud = async (opts) => {
         cloud,
         region,
         type,
-        gpu: gpu.replace(/^tesla$/, 'v100'),
+        gpu: gpu === 'tesla' ? 'v100' : gpu,
         hdd_size,
         ssh_private,
         spot,

